### PR TITLE
Fix base url

### PR DESCRIPTION
**Pull request by gal-berger**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/22837

## Description
DEMISTO_BASE_URL slash inconsistency

## Must have
- [x] Unit Test or Example Code
- [x] Changelog entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 ## 2.0.9
-Improved error message when missing authentication parameters.
+* Improved error message when missing authentication parameters.
+* fix base_url slash inconsistency
```
Comment (DeanArbel) [inline CHANGELOG.md]: It's configuration.host and not base_url

Comment (gal-berger) [reply CHANGELOG.md]: sure, fixing it.

Comment (gal-berger) [reply CHANGELOG.md]: I tested generic_request_func directly with the next options:
1. base/ + /path
2.base + / path
3. base + path
4.base/ + path

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 [1]: https://pypi.org/project/demisto-py/#history
 ## 2.0.9
-Improved error message when missing authentication parameters.
+* Improved error message when missing authentication parameters.
```
Comment (glicht) [inline CHANGELOG.md]: Add the change log entries to a new version: `2.0.10`. 2.0.9 was released alread.

Comment (gal-berger) [reply CHANGELOG.md]: done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,8 +3,11 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+## 2.0.10
+* fix configuration.host slash inconsistency
```
Comment (amshamah419) [inline CHANGELOG.md]:
```suggestion
* Enabled host and path parameters to function with trailing or leading slashes.
```

Comment (gal-berger) [reply CHANGELOG.md]: done

**Pull request merged**